src/net-socket/lib/client-socket-message.ts

```diff
@@ -0,0 +1,173 @@
+import '../helpers/dotenv-init.js';
+import * as net from 'node:net';
+import type { ErrorNet, ErrType, MessageToServer } from '../types/net-socket-types.js';
+import {
+  JobSrvQueuePrintType,
+  MESSAGE_SEPARATOR,
+  ServerResponce,
+  RecievedServerMessages,
+  TBaseResultJob,
+} from '../types/net-socket-types.js';
+import { splitMessages } from '../helpers/socket-helpers.js';
+import { ILogger } from '../../logger/logger.interface.js';
+
+// export class SocketMessagingClient<T extends TBaseResultJob> {
+export class SocketMessagingClient {
+  clientSocket: net.Socket;
+
+  recievedServerMessages: Array<RecievedServerMessages<any>> = [];
+  private msgBuffer = '';
+  currentQueryIndex = 0;
+  /*
+   * clientWaitForServerAnswer - максимальное время ожидания клиентом ответа от сервера
+   */
+  constructor(public name: string, public log: ILogger, public clientWaitForServerAnswer = 20_000) {}
+
+  /*
+   * connect
+   *
+   * извлекает параметры из .env файла:
+   * SOCKET_HOST SOCKET_PORT
+   */
```

Comment:
```suggestion
  /**
   * connect
   *
   * извлекает параметры из .env файла:
   * SOCKET_HOST SOCKET_PORT
   */
```

Лучше использовать формат jsdoc